front/pages/edit.vue

```diff
@@ -37,20 +39,71 @@
             h-100
             justify-between
           "
+          style="max-width: 500px"
         >
-            <div class="flex flex-1 m-4 justify-center">
-              <AppButton
-                size="medium"
-                v-on:click="Overlay()"
-                class="text-white my-auto"
-                :style="myPixelButtonStyle"
-              >
-                Votre pixel: {{ x + 1 }}:{{ y + 1 }}
-              </AppButton>
+          <div @click="showHelp = true" class="mr-0 ml-auto"><img src="https://img.icons8.com/ios/50/000000/help.png"/></div>
```

Comment:
[this](https://github.com/Sorikairox/micronation/pull/29#discussion_r729893109) still happens